bin/cli.py

```diff
@@ -19,98 +22,46 @@ def train(
     name: str = typer.Option(
         ..., "-n", "--name", help="Name of experiment to save under."
     ),
+    epochs: int = typer.Option(
+        ..., "-epochs", help="Number of epochs to run."
+    ),
+    batch_size: int = typer.Option(
+        ..., "-batch_size", help="Data batch size."
+    ),
+    learning_rate: float = typer.Option(
+        ..., "-learning_rate", help="Training learning rate."
+    ),
 ):
     print(f"Running experiment {name}")
     device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
-    epochs = 2
-    batch_size = 1000
-    learning_rate = 0.01
 
     # save the parameters!
 
     # load model
-    model = Net().to(device)
+    model = ser.model.Net().to(device)
 
     # setup params
     optimizer = optim.Adam(model.parameters(), lr=learning_rate)
```

Comment:
Is this really part of the cli app? or is it intrinsic to the training model setup? Genuine question :)

Reply:
This is a part of the training model. We instantiate a model that is defined in her/model as class Net

Reply:
Maybe it is better to put it in with the model then? or perhaps in a model setup utility?